app/assets/javascripts/discourse/models/category.js.es6

```diff
@@ -44,6 +45,11 @@ const Category = RestModel.extend({
     return { type: "category", id, category: this };
   },
 
+  @computed("notification_level")
+  isMuted(notification_level) {
```

Comment:
I will usually change this to `notificationLevel` when I can, we are receiving a "ruby" formatted value, but we are now in js world so I will transform it. I see it as a boundary between two world.